src/packages/src/xroad/ubuntu/generic/control

```diff
@@ -30,7 +30,7 @@ Description: X-Road central server
 Package: xroad-base
 Architecture: amd64
 Pre-depends: dpkg (>= 1.15.7.2)
-Depends: ${misc:Depends}, openjdk-8-jre-headless | java8-runtime-headless | java8-runtime, rlwrap, ca-certificates-java, crudini, adduser, net-tools, iproute2, sudo, openssl
```

Comment:
I think it would simplify things if you keep this declaration (`openjdk-8-jre-headless | java8-runtime-headless | java8-runtime` on xroad-base, but add the `openjdk-11-jre-headless | java11-runtime-headless | java11-runtime` just to xroad-proxy. That would force java 11 when installing the security server,  but use java 8 otherwise (java11 provides  also the "java8-runtime-headless" virtual package, so the dependency is satisfied).

Reply:
Alright I think I got it now. Only xroad-proxy depends on jre11 now so it gets installed only when security server is installed.